[CI] Site Build Warnings: Page Templates only allow default export of a valid React component named exports of a page query

### Current Behavior
```
warn
/Users/l/code/layer5/src/sections/Projects/Sistent/components/select/index.js
  227:1  warning  In page templates only a default export of a valid React component and the named exports of a page query,
 getServerData, Head or config are allowed.
        All other named exports will cause Fast Refresh to not preserve local component state and do a full refresh.

        Please move your other named exports to another file. Also make sure that you only export page queries that use the
 "graphql" tag from "gatsby".
  limited-exports-page-templates
  262:1  warning  In page templates only a default export of a valid React component and the named exports of a page query,
 getServerData, Head or config are allowed.
        All other named exports will cause Fast Refresh to not preserve local component state and do a full refresh.

        Please move your other named exports to another file. Also make sure that you only export page queries that use the
 "graphql" tag from "gatsby".
  limited-exports-page-templates

✖ 2 problems (0 errors, 2 warnings)
```

### Desired Situation
Warnings resolved.

---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

--- a/src/sections/Projects/Sistent/components/select/code.js
+++ b/src/sections/Projects/Sistent/components/select/code.js
@@ -15,7 +15,10 @@ import {
   OutlinedInput,
   Select,
 } from "@mui/material";
-import { Header, SectionNav } from ".";
+
+import { useLocation } from "@reach/router";
+import { navigate } from "gatsby";
+import TabButton from "../../../../../reusecore/Button";
 
 const codes = {
   "variant-outlined": `<Select>
@@ -152,11 +155,45 @@ const SelectCode = () => {
     let agesList = event.target.value;
     setMultipleAges(agesList);
   };
+  const location = useLocation();
   return (
     <SistentLayout title="Select">
       <div className="content">
-        <Header />
-        <SectionNav />
+        <a id="Identity">
+          <h2>Select</h2>
+        </a>
+        <p>
+        Select component is a dropdown menu for selecting an option from a list.
+        </p>
+        <div className="filterBtns">
+          <TabButton
+            className={
+              location.pathname === "/projects/sistent/components/select"
+                ? "active"
+                : ""
+            }
+            onClick={() => navigate("/projects/sistent/components/select")}
+            title="Overview"
+          />
+          <TabButton
+            className={
+              location.pathname === "/projects/sistent/components/select/guidance"
+                ? "active"
+                : ""
+            }
+            onClick={() => navigate("/projects/sistent/components/select/guidance")}
+            title="Guidance"
+          />
+          <TabButton
+            className={
+              location.pathname === "/projects/sistent/components/select/code"
+                ? "active"
+                : ""
+            }
+            onClick={() => navigate("/projects/sistent/components/select/code")}
+            title="Code"
+          />
+        </div>
 
         <div className="main-content">
           <a id="Select variants">

--- a/src/sections/Projects/Sistent/components/select/code.js
+++ b/src/sections/Projects/Sistent/components/select/code.js
@@ -15,10 +15,8 @@ import {
   OutlinedInput,
   Select,
 } from "@mui/material";
-
-import { useLocation } from "@reach/router";
-import { navigate } from "gatsby";
-import TabButton from "../../../../../reusecore/Button";
+import SectionNav from "./section-nav";
+import Header from "./header";
 
 const codes = {
   "variant-outlined": `<Select>
@@ -155,45 +153,11 @@ const SelectCode = () => {
     let agesList = event.target.value;
     setMultipleAges(agesList);
   };
-  const location = useLocation();
   return (
     <SistentLayout title="Select">
       <div className="content">
-        <a id="Identity">
-          <h2>Select</h2>
-        </a>
-        <p>
-        Select component is a dropdown menu for selecting an option from a list.
-        </p>
-        <div className="filterBtns">
-          <TabButton
-            className={
-              location.pathname === "/projects/sistent/components/select"
-                ? "active"
-                : ""
-            }
-            onClick={() => navigate("/projects/sistent/components/select")}
-            title="Overview"
-          />
-          <TabButton
-            className={
-              location.pathname === "/projects/sistent/components/select/guidance"
-                ? "active"
-                : ""
-            }
-            onClick={() => navigate("/projects/sistent/components/select/guidance")}
-            title="Guidance"
-          />
-          <TabButton
-            className={
-              location.pathname === "/projects/sistent/components/select/code"
-                ? "active"
-                : ""
-            }
-            onClick={() => navigate("/projects/sistent/components/select/code")}
-            title="Code"
-          />
-        </div>
+        <Header />
+        <SectionNav />
 
         <div className="main-content">
           <a id="Select variants">

--- a/src/sections/Projects/Sistent/components/select/guidance.js
+++ b/src/sections/Projects/Sistent/components/select/guidance.js
@@ -13,10 +13,8 @@ import {
   OutlinedInput,
 } from "@mui/material";
 import { useStyledDarkMode } from "../../../../../theme/app/useStyledDarkMode";
-import { useLocation } from "@reach/router";
-import { navigate } from "gatsby";
-import TabButton from "../../../../../reusecore/Button";
-
+import SectionNav from "./section-nav";
+import Header from "./header";
 const SelectGuidance = () => {
   const { isDark } = useStyledDarkMode();
   const [selectedAge, setSelectedAge] = React.useState("");
@@ -25,45 +23,11 @@ const SelectGuidance = () => {
     let agesList = event.target.value;
     setMultipleAges(agesList);
   };
-  const location = useLocation();
   return (
     <SistentLayout title="Select">
       <div className="content">
-        <a id="Identity">
-          <h2>Select</h2>
-        </a>
-        <p>
-        Select component is a dropdown menu for selecting an option from a list.
-        </p>
-        <div className="filterBtns">
-          <TabButton
-            className={
-              location.pathname === "/projects/sistent/components/select"
-                ? "active"
-                : ""
-            }
-            onClick={() => navigate("/projects/sistent/components/select")}
-            title="Overview"
-          />
-          <TabButton
-            className={
-              location.pathname === "/projects/sistent/components/select/guidance"
-                ? "active"
-                : ""
-            }
-            onClick={() => navigate("/projects/sistent/components/select/guidance")}
-            title="Guidance"
-          />
-          <TabButton
-            className={
-              location.pathname === "/projects/sistent/components/select/code"
-                ? "active"
-                : ""
-            }
-            onClick={() => navigate("/projects/sistent/components/select/code")}
-            title="Code"
-          />
-        </div>
+        <Header />
+        <SectionNav />
         <div className="main-content">
           <a id="Functions">
             <h2>Functions</h2>

--- a/src/sections/Projects/Sistent/components/select/guidance.js
+++ b/src/sections/Projects/Sistent/components/select/guidance.js
@@ -13,7 +13,10 @@ import {
   OutlinedInput,
 } from "@mui/material";
 import { useStyledDarkMode } from "../../../../../theme/app/useStyledDarkMode";
-import { Header, SectionNav } from ".";
+import { useLocation } from "@reach/router";
+import { navigate } from "gatsby";
+import TabButton from "../../../../../reusecore/Button";
+
 const SelectGuidance = () => {
   const { isDark } = useStyledDarkMode();
   const [selectedAge, setSelectedAge] = React.useState("");
@@ -22,11 +25,45 @@ const SelectGuidance = () => {
     let agesList = event.target.value;
     setMultipleAges(agesList);
   };
+  const location = useLocation();
   return (
     <SistentLayout title="Select">
       <div className="content">
-        <Header />
-        <SectionNav />
+        <a id="Identity">
+          <h2>Select</h2>
+        </a>
+        <p>
+        Select component is a dropdown menu for selecting an option from a list.
+        </p>
+        <div className="filterBtns">
+          <TabButton
+            className={
+              location.pathname === "/projects/sistent/components/select"
+                ? "active"
+                : ""
+            }
+            onClick={() => navigate("/projects/sistent/components/select")}
+            title="Overview"
+          />
+          <TabButton
+            className={
+              location.pathname === "/projects/sistent/components/select/guidance"
+                ? "active"
+                : ""
+            }
+            onClick={() => navigate("/projects/sistent/components/select/guidance")}
+            title="Guidance"
+          />
+          <TabButton
+            className={
+              location.pathname === "/projects/sistent/components/select/code"
+                ? "active"
+                : ""
+            }
+            onClick={() => navigate("/projects/sistent/components/select/code")}
+            title="Code"
+          />
+        </div>
         <div className="main-content">
           <a id="Functions">
             <h2>Functions</h2>

--- a/src/sections/Projects/Sistent/components/select/header.js
+++ b/src/sections/Projects/Sistent/components/select/header.js
@@ -1,0 +1,15 @@
+import React from "react";
+
+function Header() {
+  return (
+    <>
+      <a id="Identity">
+        <h2>Select</h2>
+      </a>
+      <p>
+        Select component is a dropdown menu for selecting an option from a list.
+      </p>
+    </>
+  );
+}
+export default Header;

--- a/src/sections/Projects/Sistent/components/select/index.js
+++ b/src/sections/Projects/Sistent/components/select/index.js
@@ -21,12 +21,46 @@ const SistentSelect = () => {
     let agesList = event.target.value;
     setMultipleAges(agesList);
   };
+  const location = useLocation();
 
   return (
     <SistentLayout title="Select">
       <div className="content">
-        <Header />
-        <SectionNav />
+        <a id="Identity">
+          <h2>Select</h2>
+        </a>
+        <p>
+        Select component is a dropdown menu for selecting an option from a list.
+        </p>
+        <div className="filterBtns">
+          <TabButton
+            className={
+              location.pathname === "/projects/sistent/components/select"
+                ? "active"
+                : ""
+            }
+            onClick={() => navigate("/projects/sistent/components/select")}
+            title="Overview"
+          />
+          <TabButton
+            className={
+              location.pathname === "/projects/sistent/components/select/guidance"
+                ? "active"
+                : ""
+            }
+            onClick={() => navigate("/projects/sistent/components/select/guidance")}
+            title="Guidance"
+          />
+          <TabButton
+            className={
+              location.pathname === "/projects/sistent/components/select/code"
+                ? "active"
+                : ""
+            }
+            onClick={() => navigate("/projects/sistent/components/select/code")}
+            title="Code"
+          />
+        </div>
         <div className="main-content">
           <p>
             The Select component is a versatile and customizable element
@@ -224,52 +258,5 @@ const SistentSelect = () => {
   );
 };
 
-export function SectionNav() {
-  const location = useLocation();
-  return (
-    <div className="filterBtns">
-      <TabButton
-        className={
-          location.pathname === "/projects/sistent/components/select"
-            ? "active"
-            : ""
-        }
-        onClick={() => navigate("/projects/sistent/components/select")}
-        title="Overview"
-      />
-      <TabButton
-        className={
-          location.pathname === "/projects/sistent/components/select/guidance"
-            ? "active"
-            : ""
-        }
-        onClick={() => navigate("/projects/sistent/components/select/guidance")}
-        title="Guidance"
-      />
-      <TabButton
-        className={
-          location.pathname === "/projects/sistent/components/select/code"
-            ? "active"
-            : ""
-        }
-        onClick={() => navigate("/projects/sistent/components/select/code")}
-        title="Code"
-      />
-    </div>
-  );
-}
-
-export function Header() {
-  return (
-    <>
-      <a id="Identity">
-        <h2>Select</h2>
-      </a>
-      <p>
-        Select component is a dropdown menu for selecting an option from a list.
-      </p>
-    </>
-  );
-}
 
 export default SistentSelect;

--- a/src/sections/Projects/Sistent/components/select/index.js
+++ b/src/sections/Projects/Sistent/components/select/index.js
@@ -2,10 +2,8 @@ import React from "react";
 import { SistentThemeProvider } from "@layer5/sistent";
 import { SistentLayout } from "../../sistent-layout";
 import { Row } from "../../../../../reusecore/Layout";
-import { useLocation } from "@reach/router";
+
 import { useStyledDarkMode } from "../../../../../theme/app/useStyledDarkMode";
-import { navigate } from "gatsby";
-import TabButton from "../../../../../reusecore/Button";
 import {
   FormControl,
   InputLabel,
@@ -13,6 +11,8 @@ import {
   Select,
   OutlinedInput,
 } from "@mui/material";
+import SectionNav from "./section-nav";
+import Header from "./header";
 
 const SistentSelect = () => {
   const { isDark } = useStyledDarkMode();
@@ -21,46 +21,12 @@ const SistentSelect = () => {
     let agesList = event.target.value;
     setMultipleAges(agesList);
   };
-  const location = useLocation();
 
   return (
     <SistentLayout title="Select">
       <div className="content">
-        <a id="Identity">
-          <h2>Select</h2>
-        </a>
-        <p>
-        Select component is a dropdown menu for selecting an option from a list.
-        </p>
-        <div className="filterBtns">
-          <TabButton
-            className={
-              location.pathname === "/projects/sistent/components/select"
-                ? "active"
-                : ""
-            }
-            onClick={() => navigate("/projects/sistent/components/select")}
-            title="Overview"
-          />
-          <TabButton
-            className={
-              location.pathname === "/projects/sistent/components/select/guidance"
-                ? "active"
-                : ""
-            }
-            onClick={() => navigate("/projects/sistent/components/select/guidance")}
-            title="Guidance"
-          />
-          <TabButton
-            className={
-              location.pathname === "/projects/sistent/components/select/code"
-                ? "active"
-                : ""
-            }
-            onClick={() => navigate("/projects/sistent/components/select/code")}
-            title="Code"
-          />
-        </div>
+        <Header />
+        <SectionNav />
         <div className="main-content">
           <p>
             The Select component is a versatile and customizable element

--- a/src/sections/Projects/Sistent/components/select/section-nav.js
+++ b/src/sections/Projects/Sistent/components/select/section-nav.js
@@ -1,0 +1,40 @@
+import React from "react";
+import { useLocation } from "@reach/router";
+import { navigate } from "gatsby";
+import TabButton from "../../../../../reusecore/Button";
+
+function SectionNav() {
+  const location = useLocation();
+  return (
+    <div className="filterBtns">
+      <TabButton
+        className={
+          location.pathname === "/projects/sistent/components/select"
+            ? "active"
+            : ""
+        }
+        onClick={() => navigate("/projects/sistent/components/select")}
+        title="Overview"
+      />
+      <TabButton
+        className={
+          location.pathname === "/projects/sistent/components/select/guidance"
+            ? "active"
+            : ""
+        }
+        onClick={() => navigate("/projects/sistent/components/select/guidance")}
+        title="Guidance"
+      />
+      <TabButton
+        className={
+          location.pathname === "/projects/sistent/components/select/code"
+            ? "active"
+            : ""
+        }
+        onClick={() => navigate("/projects/sistent/components/select/code")}
+        title="Code"
+      />
+    </div>
+  );
+}
+export default SectionNav;

